ci/build-test-ios.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e -u -o -x pipefail
+
+cd "$(dirname "$0")/../"
+
+# Get shared CI and prepare unity
+ci/bootstrap.sh
+.shared-ci/scripts/prepare-unity.sh
+
+ci/test.sh
+ci/build-ios_client.sh
```

Comment:
Not sure if copying the script is best solution. Considering that we will have Android client build as well, converting this to parameterised script looks slightly better idea to me.
What do people think?

Reply:
I'd like to add a generic one to https://improbableio.atlassian.net/browse/UTY-986 but haven't got around to it 🤷‍♂️

Reply:
See spatialos/gdk-for-unity-shared-ci#2 for this generic build script.

Also no idea why that JIRA link was copy pasted 😂